chassis/configuration/configuration/master/master.conf.xml

```diff
@@ -22,9 +22,12 @@
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:contest="http://www.springframework.org/schema/context"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
        http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd">
 
+     <!--to include user defined Jagger Load Test scenarios-->
+    <contest:component-scan base-package="com.griddynamics.jagger.config"/>
```

Comment:
it still works because of this 
` xmlns:contest="http://www.springframework.org/schema/context"`
but will rename